public/app.js

```diff
@@ -24,43 +25,52 @@ $('li.navigation:eq(0)').on('click', function(){
 $('li.navigation:eq(1)').on('click', function(){
   $('.content').hide();
   $('.main').appendTo('body');
-})
-
-function Project (projectData) {
-  this.title = projectData.title;
-  this.projectUrl = projectData.projectUrl;
-  this.projectImg = projectData.projectImg;
-  this.description = projectData.description;
-}
+});
 
-Project.prototype.toHtml = function () {
-  var sourceHTML = $('#newTemplate').html();
-  var actualTemplate = Handlebars.compile(sourceHTML);
-  var newRawHTML = actualTemplate(this);
-  return actualTemplate(this);
-}
+(function(module){
+  function Project (projectData) {
+    this.title = projectData.title;
+    this.projectUrl = projectData.projectUrl;
+    this.projectImg = projectData.projectImg;
+    this.description = projectData.description;
+  }
 
-Project.loadAll = function(projectData){
+  Project.all = [];
 
-  projectData.forEach(function(projectObject) {
-    Project.all.push(new Project(projectObject));
-  });
+  Project.Title = () => {
+    return Project.all.map(ele => ele.title)
+                      .reduce((aggregator, title) => {
```

Comment:
This is redundant with the `map` on line 41 since the `map` already will return an array of titles.